flink-runtime/src/test/java/org/apache/flink/runtime/io/network/partition/consumer/SingleInputGateTest.java

```diff
@@ -517,21 +514,22 @@ public void testUpdateUnknownInputChannel() throws Exception {
 			assertThat(inputGate.getInputChannels().get(localResultPartitionId.getPartitionId()),
 				is(instanceOf((UnknownInputChannel.class))));
 
+			ResourceID localLocation = new ResourceID("local");
```

Comment:
Could be factored out as a constant

Reply:
I think it would be better to use random `ResourceID.generate()`